FirebaseAppCheck/Sources/AppAttestProvider/API/FIRAppAttestAPIService.m

```diff
@@ -30,12 +30,19 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-static NSString *const kRequestFieldAttestation = @"attestation_statement";
-static NSString *const kRequestFieldKeyID = @"key_id";
+static NSString *const kRequestFieldArtifact = @"artifact";
+static NSString *const kRequestFieldAssertion = @"assertion";
+static NSString *const kRequestFieldAttestation = @"attestationStatement";
```

Comment:
This is something we will need to double check because the DeviceCheck endpoint [accepts snake case field names](https://github.com/firebase/firebase-ios-sdk/blob/44ab89ec8d9ad817bf6fded3144e829a32ef9f10/FirebaseAppCheck/Sources/DeviceCheckProvider/API/FIRDeviceCheckAPIService.m#L40).